src/config/Widgets/widgets.js

```diff
@@ -77,6 +78,7 @@ const getItaliaWidgets = (config) => {
         [],
       ),
       subsite_social_links: SubsiteSocialLinksWidget,
+      canale_digitale: (props) => <CanaleDigitaleWidget {...props} />,
```

Comment:
```suggestion
      canale_digitale: CanaleDigitaleWidget,
```

Se non erro è sufficiente mettere direttamente il componente come ho ipotizzato qui nel suggerimento